core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 	"time"
 
+	sq "github.com/Masterminds/squirrel"
```

Comment:
Is this important to have? It doesn't look like we are using it for anything fancy, and it comes with some extra dependencies. Could we use regular SQL instead?

Reply:
I can imagine some interesting uses though, and @krehermann might want to take a look

Reply:
Yes,  it's possible to implement a workaround, but it will be a bit messier. 
Let me know, if you'd like me to drop this dep

Reply:
I didn't see all the logic at first, so I can see how it is helpful now, but for me it is still a little difficult to read and it seems inefficient too. Especially when reusing for the second query.